ls8/cpu.c

```diff
@@ -125,10 +118,10 @@ void cpu_run(struct cpu *cpu)
   while (running) {
     // TODO
     // 1. Get the value of the current instruction (in address PC).
-    unsigned char IR = cpu->ram[cpu->PC];
+    unsigned char IR = cpu_ram_read(cpu, cpu->PC);
     
     // 2. Figure out how many operands this next instruction requires
-    int num_of_operands = num_operands_needed(IR);
+    int num_of_operands = num_operands_needed(IR); 
```

Comment:
personal preference, but I feel like adding a function call here is making tings a bit more complicated